parser/internal/parserv2/operators.go

```diff
@@ -13,6 +13,7 @@ func isPrefixOperator(token ast.Token) bool {
 
 func isInfixOperator(token ast.Token) bool {
 	return token == ast.Equals ||
+		token == ast.EqualSign ||
```

Comment:
I assume this one comes from `parservX`, so maybe this is the "wrong" place to review, _but_: I don't think this is needed. We don't treat it as an infix operator anywhere 🤔